whisper/src/rpc/crypto.rs

```diff
@@ -106,12 +107,14 @@ impl EncryptionInstance {
 					}
 					AesEncode::OnTopics(topics) => {
 						let mut buf = Vec::new();
-						let key = H256(key);
+						let key = H256(key.into());
 
 						for topic in topics {
 							buf.extend(&*(topic ^ key));
 						}
 
+						let _ = Memzero::from(key.0);
```

Comment:
doesn't that just copy the value of `key` into some other place in the stack and then zero that?

Reply:
Yes, thanks.